client/my-sites/site-settings/site-owner-transfer/pending-domain-transfer.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from '@automattic/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { sprintf } from '@wordpress/i18n';
+import { useTranslate } from 'i18n-calypso';
+import ActionPanelBody from 'calypso/components/action-panel/body';
+import ActionPanelFooter from 'calypso/components/action-panel/footer';
+import { ResponseDomain } from 'calypso/lib/domains/types';
+
+const PendingDomainTransfer = ( { domain }: { domain: ResponseDomain } ) => {
+	const translate = useTranslate();
+	return (
+		<>
+			<ActionPanelBody>
+				<p>
+					{ createInterpolateElement(
+						sprintf(
+							// translators: %s is the domain name
+							translate(
+								'There are pending domain transfers for <strong>%s</strong>. Please, complete them before transferring the site.'
```

Comment:
Let's remove the comma after `please` and use this instead: 
```suggestion
								'There are pending domain transfers for <strong>%s</strong>. Please complete them before transferring the site.'
```
I corrected the Figma file as well.